deploy-gae-build-task/test/mocha.opts

```diff
@@ -0,0 +1,3 @@
+﻿--require ts-node/register
```

Comment:
Is this config file for mocha that mocha reads it by default,  just like web.config to ASP.NET?